packages/code-du-travail-api/src/server/routes/search/search.sem.js

```diff
@@ -0,0 +1,26 @@
+function getSemQuery({ query_vector, size, sources = [] }) {
+  return {
+    size: size,
+    _source: ["title", "source", "slug", "anchor", "url"],
+    query: {
+      script_score: {
+        query: {
+          bool: {
+            filter: {
+              bool: {
+                should: sources.map(source => ({ term: { source } }))
```

Comment:
what if sources is empty ?

Reply:
don't really know :) I will test that

Reply:
source should not be empty, I will throw an error